scripts/quests/jeuno/LB06_New_Worlds_Await.lua

```diff
@@ -28,10 +28,10 @@ quest.sections =
     {
         check = function(player, status, vars)
             return status == QUEST_AVAILABLE and
+                xi.settings.MAX_LEVEL >= 80 and
```

Comment:
This step handles giving Limit Breaker KI. Wich requires lvl limit of 75, not 80

Reply:
well shit

Reply:
I dunno wtf did it then. I have not touched anything and my settings have had 75 for a long time.